examples/hono/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hono",
+  "type": "module",
+  "scripts": {
+    "dev": "tsx watch src/index.ts",
```

Comment:
Should this be `vc dev`? Maybe it shouldn't because we can't assume the user has `vc` installed?

Reply:
yeah `vc dev` is the recommendation for now